src/test/java/org/opensrp/repository/postgres/ClientsRepositoryTest.java

```diff
@@ -39,6 +39,7 @@
 	protected Set<String> getDatabaseScripts() {
 		Set<String> scripts = new HashSet<String>();
 		scripts.add("client.sql");
+		scripts.add("event.sql");
```

Comment:
Hi @githengi  @dubdabasoduba,
There were some unit tests failing on ClientRepoTest, after investigation I have figured out that few tests are also dependent on event tables as well. Therefore, we need to add a script of `event.sql` in the `ClientsRepositoryTest`. Please review!

Moreover, I also tried to revert to the previous state of clients.sql but it did not work. The problem was resolved after adding events.sql in the scripts section!